docs/addons/gmaestro.md

```diff
@@ -0,0 +1,100 @@
+# gMaestro AddOn for Amazon EKS Blueprints
+
+This repository contains the source code for the gMaestro AddOn for [Amazon EKS Blueprints](https://aws-quickstart.github.io/cdk-eks-blueprints/). This AddOn is a [CDK](https://aws.amazon.com/cdk/) construct that allows customers to add gMaestro to their Amazon EKS clusters.
+
+gMaestro is a Kubernetes cost optimization solution that helps companies reduce spending on unutilized resources by up to 60%.
+With gMaestro, you gain full visibility into K8s clusters, seamlessly interact with HPA scaling policies, and achieve your cost-performance goals by applying custom rightsizing recommendations based on actual usage in production.
+1. **Install gMaestro** - As a single pod on your cluster with a single command line.
+2. **Multi-cluster visibility** - Gain visibility into each controller, resource request, container, HPA policy, utilization, and cost.
+3. **Apply custom recommendations** - gMaestro generates custom recommendations by analyzing the actual utilization of each controller.
+
+For additional information, visit [gMaestro documentation](https://gmaestro.gitbook.io/gmaestro-docs/).
+
+## Prerequisite 
+Before using gMaestro, you need to:
+1. [Sign up](https://app.granulate.io/gMaestroSignup) to the gMaestro platform
+2. Download a sample YAML file - After signing up to gMaestro, navigate to the [Deploy](https://app.granulate.io/deploy) on the left-hand menu, fill in the required fields and click on "Generate Config File" 
+
+
+## Installation
+
+Using [npm](https://npmjs.org):
+
+```bash
+$ npm i @granulate/gmaestro-eks-blueprints-addon
+```
+
+## Usage
+
+```typescript
+import 'source-map-support/register';
+import * as CDK from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import {GmaestroAddOn} from '@granulate/gmaestro-eks-blueprints-addon';
+
+const app = new cdk.App();
+
+const addOn = new GmaestroAddOn({
+        clientIdSecretName: "<client id secret name>", // Create and copy from gMaestro deployment yaml
+        clientName: "<client name>",
+        clusterName: "<cluster name>",
+        namespace: "<namespace>",
+        grafanaMetricsSecretName: "<grafana metrics secret name>", // Create and copy from gMaestro deployment yaml
```

Comment:
I am unclear what "Create and copy from gMaestro deployment yaml" means. Can you clarify in the doc before the snippet?

Reply:
done